registry/server/migrations/20210405182655_routes_domainId.ts

```diff
@@ -1,7 +1,8 @@
 import { Knex } from 'knex';
+import { isSqlite } from '../util/db';
 
 export async function up(knex: Knex): Promise<any> {
-    if (isMySQL(knex)) {
+    if (!isSqlite(knex)) {
```

Comment:
Maybe it makes sense to remove SqlLite at all to not support this logical branch ?